rllib/core/rl_module/torch/torch_marl_module.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any, Mapping
+from ray.rllib.utils.framework import try_import_torch
+from ray.rllib.core.rl_module import MultiAgentRLModule
+from ray.rllib.core.rl_module.torch.torch_rl_module import TorchRLModule
+
+
+torch, nn = try_import_torch()
+
+
+class TorchMARLModule(MultiAgentRLModule, nn.Module):
```

Comment:
`TorchMultiAgentRLModule`

Sorry, I know these names are long, but it's super nice for a user to look at classnames that are immediately easy to read and understand, even if they are long. Your IDE will help you here anyways.

Reply:
fixed.